cmd/charm/charmcmd/show.go

```diff
@@ -284,6 +284,10 @@ func (s *showData) formatTabular() {
 }
 
 func (s *showData) printChannels() {
+	if len(s.channels) == 0 {
+		// Unpublished charm
+		return
```

Comment:
I wonder if instead of leaving the user without any info about the channels, we would rather explicitly say "none" or "not released".